lib/dynamo/dynamo/jobs.py

```diff
@@ -2,23 +2,8 @@
 from typing import List
 
 from boto3.dynamodb.conditions import Attr, Key
-from dateutil.parser import parse
 
-from hyp3_api import DYNAMODB_RESOURCE
-from hyp3_api.util import format_time
-
-
-def get_request_time_expression(start, end):
-    key = Key('request_time')
-    formatted_start = (format_time(parse(start))if start else None)
-    formatted_end = (format_time(parse(end)) if end else None)
-
-    if formatted_start and formatted_end:
-        return key.between(formatted_start, formatted_end)
-    if formatted_start:
-        return key.gte(formatted_start)
-    if formatted_end:
-        return key.lte(formatted_end)
+from dynamo.util import DYNAMODB_RESOURCE, get_request_time_expression
 
 
 def put_jobs(payload: List[dict]):
```

Comment:
I think we should move the business logic to add the request_time, status_code, and job_id fields into this function, so we don't have to duplicate it between the `api` and `process_new_granules`. That might move a few additional functions from api.util to dynamo.util.

Reply:
took a stab at this in https://github.com/ASFHyP3/hyp3/pull/512 , which is ready for review